### PR TITLE
Added error handling to statcast functions. Added read.csv to Namespa…

### PR DESCRIPTION
**Pull request by keberwein**
**Additions**
Added error handling to the Statcast functions. I put a try/catch around the read.csv function with some custom error messages. This way the user will know exactly what they did wrong instead of a generic error. 
Added `imortFrom utils read.csv` to Namespace, it was missing for some reason. 

**R CMD check error fixes:**
Check errored on DESCRIPTION, the word "Author" was malformed.
Check threw notes on `scrape_statcast_savant_pitcher` and `scrape_statcast_savant_batter` 
becuase the example lines were over 100 characters. Easy fix, inserted breaks.

### CODE REVIEW

File: R/scrape_statcast_savant_batter.R [hunk 1]
```diff
@@ -5,30 +5,78 @@
...
-  x$hit_speed <- as.character(x$hit_speed) %>% as.numeric()
-  x$hit_angle <- as.character(x$hit_angle) %>% as.numeric()
-  x$effective_speed <- as.character(x$effective_speed) %>% as.numeric()
-  x$release_spin_rate <- as.character(x$release_spin_rate) %>% as.numeric()
-  x$release_extension <- as.character(x$release_extension) %>% as.numeric()
-  x$barrel <- with(x, ifelse(hit_angle <= 50 & hit_speed >= 98 & hit_speed * 1.5 - hit_angle >= 11 & hit_speed + hit_angle >= 124, 1, 0))
-  x
+    # Check to make sure args are in the correct format.
+    if(!is.character(start_date) | !is.character(end_date)) {
+        warning("Please wrape your dates in quotations in 'yyyy-mm-dd' format.")
```
Comment (BillPetti) [inline R/scrape_statcast_savant_batter.R]: Looks like a typo: `wrape` should be `wrap`

File: R/scrape_statcast_savant_pitcher.R [hunk 2]
```diff
@@ -5,31 +5,80 @@
...
-  x$hit_speed <- as.character(x$hit_speed) %>% as.numeric()
-  x$hit_angle <- as.character(x$hit_angle) %>% as.numeric()
-  x$effective_speed <- as.character(x$effective_speed) %>% as.numeric()
-  x$release_spin_rate <- as.character(x$release_spin_rate) %>% as.numeric()
-  x$release_extension <- as.character(x$release_extension) %>% as.numeric()
-  x$barrel <- with(x, ifelse(hit_angle <= 50 & hit_speed >= 98 & hit_speed * 1.5 - hit_angle >= 11 & hit_speed + hit_angle >= 124, 1, 0))
-  x
+    # Check to make sure args are in the correct format.
+    if(!is.character(start_date) | !is.character(end_date)) {
+        warning("Please wrape your dates in quotations in 'yyyy-mm-dd' format.")
```
Comment (BillPetti) [inline R/scrape_statcast_savant_pitcher.R]: `wrape` should be `wrap`

File: R/scrape_statcast_savant_pitcher.R [hunk 3]
```diff
@@ -5,31 +5,80 @@
...
+        return(NULL)
+    }
+    # Check for other user errors.
+    if(as.Date(start_date)<="2015-03-01") { # March 1, 2015 was the first dat of Spring Training.
+        message("Some metrics such as Exit Velocity and Batted Ball Events have only been compiled since 2015.")
+    }
+    if(as.Date(start_date)<="2008-03-25") { # March 25, 2008 was the first dat of Spring Training.
+        stop("The data are limited ot the 2008 MLB season and after.")
```
Comment (BillPetti) [inline R/scrape_statcast_savant_pitcher.R]: `ot` should be `to`

**Comment by keberwein** [comment]
Very nice. I fixed the typos. I'm working on a more elaborate function that will accept any (or all) of the arguments that are accepted by the BaseballSevant interface. I'll keep you posted!

**Pull request merged**